examples/query/src/main.py

```diff
@@ -1,5 +1,40 @@
-from kybra import query, text
+from kybra import opt, query, Record, update, text
+from typing import TypedDict
 
 @query
 def simple_query() -> text:
     return 'This is a query function'
+
```

Comment:
Why is all of this here?

Reply:
None of  that should be there, it's not like that on my branch

Reply:
Yeah, I put it there, and I know why it's there. I guess the comment would be better said "Why did I leave this in here?" and the answer would probably be I forgot to take it out...